project/apps/auth-service/src/domain/ports/auth.service.ts

```diff
@@ -56,17 +57,33 @@ export class AuthService {
     }
   }
 
+  async refreshUserSession(refresh_token: string): Promise<Session> {
+    try {
+      const userSession =
+        await this.authRepository.refreshUserSession(refresh_token);
+
+      this.logger.log(`user refreshed successfully`);
```

Comment:
i think we should log the user details here as well, i.e. which user has their token refreshed successfully